data/lib/compat/compat.lua

```diff
@@ -359,6 +364,13 @@ function getAccountNumberByPlayerName(name)
 	end
 	return 0
 end
+function getPlayerFlagValue(cid, flag)
+	local player = Player(cid)
+	if player == nil then
+		return false
+	end
+	return bit.band(player:getGroup():getFlags(), flag) == flag and true
```

Comment:
I'd put that because I wanted to return only the boolean, but I do not know if it is effective.
At least, in one try of debug printing, it has made difference. I will remove it.